corehq/motech/repeaters/repeater_generators.py

```diff
@@ -470,9 +492,19 @@ def get_case_updates(self):
             for prop in update_props
         }
 
-    def get_case_index(self, target_case):
+    def get_case_indices(self, target_case):
+        indices = self.get_create_case_index(target_case)
+        if not self.index_remove_case_id:
+            return indices
+
+        if self.index_remove_identifier not in indices:
```

Comment:
Is this to handle a scenario where an update attempts to both delete and create the same index?  Ie, they might think they're removing the old index and adding a new one, so the add wins?

Reply:
looks like that's what `test_generator_update_create_and_remove_same_index` illustrates.